### PR TITLE
"Get array setting values by name" return correct value

### PR DESCRIPTION
**Pull request by anivirtoway**

### CODE REVIEW

File: VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs [hunk 1]
```diff
@@ -94,7 +93,21 @@ public async Task<ActionResult> UpdateAsync([FromBody] ObjectSettingEntry[] obje
         [ApiExplorerSettings(IgnoreApi = true)]
         public async Task<ActionResult<object[]>> GetArrayAsync(string name)
         {
-            var result = (await _settingsManager.GetObjectSettingAsync(name)).AllowedValues;
+            var settings = await _settingsManager.GetObjectSettingAsync(name);
```
Comment (artem-dudarev) [inline VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs]: Rename to `setting`

File: VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs [hunk 2]
```diff
@@ -94,7 +93,21 @@ public async Task<ActionResult> UpdateAsync([FromBody] ObjectSettingEntry[] obje
...
+            object[] result;
```
Comment (artem-dudarev) [inline VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs]: Make this line a first line in the method and separate it with an empty line from the remaining code.

Comment (anivirtoway) [reply VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs]: fixed

File: VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs (hunk 1 above)
Comment (anivirtoway) [reply VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs]: fixed

**Comment by tatarincev** [comment]
This issue has been fixed in this commit https://github.com/VirtoCommerce/vc-platform-core/commit/7b6d1b2b6d1583b06e0a37c273bed4cb0e899fab

**Pull request closed without merging**